plugins/dotnet-auth-core-identity/src/core/create-static-file-map.ts

```diff
@@ -7,20 +7,24 @@ export async function createStaticFileFileMap(
   destPath: string,
   filePath: string,
   context: dotnetTypes.DsgContext,
-  classReference?: ClassReference
+  classReferences?: ClassReference[]
 ): Promise<FileMap<CodeBlock>> {
   const fileMap = new FileMap<CodeBlock>(context.logger);
 
   if (!context.resourceInfo) return fileMap;
   const resourceName = pascalCase(context.resourceInfo.name);
   let fileContent = await readFile(filePath, "utf-8");
   fileContent = fileContent.replace("ServiceName", resourceName);
+  fileContent = fileContent.replace(
+    "ServiceNameDbContext",
+    `${resourceName}DbContext`
+  );
```

Comment:
Try to use constants to define the KEYs to replace for a better visibility. 
Moreover, to replace all occurrences, use `.replaceAll` instead of `.replace`

Reply:
@overbit why? I have only one time this placeholder in the file??..

Reply:
What I meant is that you can replace all instances of `ServiceName` (or better `${{SERVICE_NAME}}`) in one got avoiding potential issues where `ServiceName` appears more than once in the static file. 

So from this
```ts
 fileContent = fileContent.replace("ServiceName", resourceName);
  fileContent = fileContent.replace(
    "ServiceNameDbContext",
    `${resourceName}DbContext`
  );
```
to this:
```ts
 fileContent = fileContent.replaceAll("ServiceName", resourceName);
```

Reply:
Done